### ISSUE TITLE
Fix some deprecation warnings

### ISSUE THREAD

**Issue by joefowler** [usnistgov/mass#218]
**[Original report](https://bitbucket.org/joe_fowler/mass/issues/218) by Joseph Fowler (Bitbucket: [joe_fowler](https://bitbucket.org/joe_fowler), ).**

----------------------------------------

`StrictVersion(”2.2.0”)` appears to be deprecated.

```text
======================================================== warnings summary =========================================================
../../../../opt/local/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/future-0.18.2-py3.9.egg/past/builtins/misc.py:45
  /opt/local/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/future-0.18.2-py3.9.egg/past/builtins/misc.py:45: DeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
    from imp import reload

../../../../opt/local/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/xraydb/utils.py:27
  /opt/local/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/xraydb/utils.py:27: DeprecationWarning: Please use `physical_constants` from the `scipy.constants` namespace, the `scipy.constants.codata` namespace is deprecated.
    R_ELECTRON_CM = 100.0 * consts.codata.physical_constants['classical electron radius'][0]

mass/core/test_core.py: 198 warnings
mass/core/test_filtering.py: 40 warnings
mass/core/test_phase_correct.py: 4 warnings
mass/regression_test/test_regression.py: 4 warnings
  /Users/fowlerj/Software/mass/mass/core/files.py:262: DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
    if StrictVersion(self.version_str.decode()) >= StrictVersion("2.2.0"):

mass/core/test_core.py: 198 warnings
mass/core/test_filtering.py: 40 warnings
mass/core/test_phase_correct.py: 4 warnings
mass/regression_test/test_regression.py: 4 warnings
  /Users/fowlerj/Software/mass/mass/core/files.py:199: DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
    if StrictVersion(self.version_str.decode()) >= StrictVersion("2.2.0"):

mass/core/test_core.py: 34 warnings
  /Users/fowlerj/Software/mass/mass/core/ljh_modify.py:199: DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
    if StrictVersion(infile.version_str.decode()) < StrictVersion("2.2.0"):

mass/off/test_channels.py::TestSummaries::test_refresh_from_files
  /opt/local/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/_pytest/unraisableexception.py:78: PytestUnraisableExceptionWarning: Exception ignored in: <function Infinite.__del__ at 0x148e993a0>
  
  Traceback (most recent call last):
    File "/opt/local/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/progress-1.6-py3.9.egg/progress/__init__.py", line 60, in __del__
      if self._hidden_cursor:
  AttributeError: 'SilenceBar' object has no attribute '_hidden_cursor'
  
    warnings.warn(pytest.PytestUnraisableExceptionWarning(msg))

-- Docs: https://docs.pytest.org/en/stable/warnings.html
```

‌

**Comment by joefowler** [comment]
**Original comment by Joseph Fowler (Bitbucket: [joe_fowler](https://bitbucket.org/joe_fowler), ).**

----------------------------------------

Replace deprecated module distutils.version

Fixes #218.

**Comment by joefowler** [comment]
**Original comment by Joseph Fowler (Bitbucket: [joe_fowler](https://bitbucket.org/joe_fowler), ).**

----------------------------------------

Removing milestone: v0.7 (automated comment)
